webserver/websocket/src/main/java/io/helidon/webserver/websocket/WsConnection.java

```diff
@@ -243,8 +248,8 @@ private boolean processFrame(ClientWsFrame frame) {
 
     private ClientWsFrame readFrame() {
         try {
-            // TODO check may payload size, danger of oom
-            return ClientWsFrame.read(ctx, dataReader, Integer.MAX_VALUE);
+            int maxFrameLength = wsConfig != null ? wsConfig.maxFrameLength() : Integer.MAX_VALUE;
```

Comment:
Default is 1Mb and default default 2Gb?

Reply:
Can the setting change between the frame reads? Could we sort it out once in the constructor?

Reply:
No, it is just that `DirectWsConnection` is not connected to `WsConfig` at this time, so it may be null. I suppose I can make it 1 MB in both cases.